charts/scalardb-cluster-monitoring/values.yaml

```
@@ -0,0 +1,693 @@
+          replacement: $1
+          action: keep
+        - source_labels: [__meta_kubernetes_service_label_app_kubernetes_io_name, __meta_kubernetes_service_labelpresent_app_kubernetes_io_name]
+          separator: ;
+          regex: (scalardb-cluster);true
+          replacement: $1
+          action: keep
+        - source_labels: [__meta_kubernetes_endpoint_port_name]
+          separator: ;
+          regex: admin
+          replacement: $1
+          action: keep
+        - source_labels: [__meta_kubernetes_endpoint_address_target_kind, __meta_kubernetes_endpoint_address_target_name]
+          separator: ;
+          regex: Node;(.*)
+          target_label: node
+          replacement: ${1}
+          action: replace
+        - source_labels: [__meta_kubernetes_endpoint_address_target_kind, __meta_kubernetes_endpoint_address_target_name]
+          separator: ;
+          regex: Pod;(.*)
+          target_label: pod
+          replacement: ${1}
+          action: replace
+        - source_labels: [__meta_kubernetes_namespace]
+          separator: ;
+          target_label: namespace
+          replacement: $1
+          action: replace
+        - source_labels: [__meta_kubernetes_service_name]
+          separator: ;
+          target_label: service
+          replacement: $1
+          action: replace
+        - source_labels: [__meta_kubernetes_pod_name]
+          separator: ;
+          target_label: pod
+          replacement: $1
+          action: replace
+        - source_labels: [__meta_kubernetes_pod_container_name]
+          separator: ;
+          target_label: container
+          replacement: $1
+          action: replace
+        - source_labels: [__meta_kubernetes_pod_phase]
+          separator: ;
+          regex: (Failed|Succeeded)
+          replacement: $1
+          action: drop
+        - source_labels: [__meta_kubernetes_service_name]
+          separator: ;
+          target_label: job
+          replacement: ${1}
+          action: replace
+        - separator: ;
+          target_label: endpoint
+          replacement: admin
+          action: replace
+        - source_labels: [__address__, __tmp_hash]
+          separator: ;
+          regex: (.+);
+          target_label: __tmp_hash
+          replacement: $1
+          action: replace
+        - source_labels: [__tmp_hash]
+          separator: ;
+          modulus: 1
+          target_label: __tmp_hash
+          replacement: $1
+          action: hashmod
+        - source_labels: [__tmp_hash]
+          separator: ;
+          regex: "0"
+          replacement: $1
+          action: keep
+        kubernetes_sd_configs:
+        - role: endpoints
+          kubeconfig_file: ""
+          follow_redirects: true
+          enable_http2: true
+          namespaces:
+            own_namespace: true
+      - job_name: kubernetes-nodes-cadvisor
+        honor_timestamps: true
+        track_timestamps_staleness: false
+        scrape_interval: 1m
```

Is it possible to modify all the scrape_interval to 10s?
```suggestion
        scrape_interval: 10s
```
When I tried it with 5s, prometheus server was CrashLoopBackOff.

`scalardb-cluster-monitoring-prometheus-server-748867c5c-gp5vf     1/2     CrashLoopBackOff   16 (82s ago)   58m`

The following message was recorded a lot.
```
level=error ts=2025-04-10T10:44:49.279342305Z caller=runutil.go:117 msg="function failed. Retrying in next tick" err="trigger reload: reload request failed: Post \"http://127.0.0.1:9090/-/reload\": dial tcp 127.0.0.1:9090: connect: connection refused"
```

---

Thank you for your review!
As we discussed in the offline meeting, let's set `10s` for all scrape_interval for now.